src/main/java/mate/academy/springintro/config/SecurityConfig.java

```diff
@@ -25,7 +30,7 @@ public PasswordEncoder getpasswordEncoder() {
     }
 
     @Bean
-    public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
+    public SecurityFilterChain securityFilterChain(HttpSecurity http, JwtAuthenticationFilter jwtAuthenticationFilter) throws Exception {
```

Comment:
Just add `private final JwtAuthenticationFilter jwtAuthFilter;` to SecurityConfig
```suggestion
    public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
```